.github/workflows/test.yml

```diff
@@ -50,3 +50,117 @@ jobs:
       if: github.event_name == 'push' && github.ref == 'refs/heads/master'
       env:
         FUZZIT_API_KEY: ${{ secrets.FUZZIT_API_KEY }}
+
+  docker:
+    name: Build and deploy Docker images
+    # Only deploy if previous stages pass.
+    needs: [test, test-linux-alpine, fuzz]
+    runs-on: ubuntu-latest
+    if: >
+      github.event_name == 'push' && 
+        (github.ref == 'refs/heads/master' ||
+         startsWith(github.ref, 'refs/tags/'))
+    env:
+      # Export environment variables for all stages.
+      DOCKER_CLI_EXPERIMENTAL: enabled  # for 'docker buildx'
+      DOCKER_BASE: mvdan/shfmt
+      DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+      # We use all platforms for which FROM images in our Dockerfile are
+      # available.
+      DOCKER_PLATFORMS: >
+        linux/386
+        linux/amd64
+        linux/arm/v6
+        linux/arm/v7
+        linux/arm64/v8
+        linux/ppc64le
+        linux/s390x
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1  # no history, just HEAD
+    - name: Set up image tags
+      run: |
+        set -vx
+        # Export environment variable for later stages.
+        if echo "$GITHUB_REF" | grep -q '^refs/heads/'; then
+          # Pushes to (master) branch - deploy 'latest'.
+          echo "::set-env name=TAGS::latest"
```

Comment:
isn't TAGS always just one word? it's probably easiest to make it singular, and remove the loops further below. if we really need multiple tags in the future, we can re-add the looping.

Reply:
Done